src/parser/wsdl.js

```diff
@@ -203,10 +228,21 @@ class WSDL {
           deepMerge(self.definitions.schemas[include.namespace ||
           wsdl.definitions.$targetNamespace], wsdl.definitions);
       }
-      self._processNextInclude(includes, function(err) {
+      self._processNextInclude(includes, function (err) {
         callback(err);
       });
-    });
+    };
+
+    if (self.syncLoad) {
+      var wsdl = WSDL.loadSync(includePath, options);
+      staticLoad(null, wsdl)
```

Comment:
`;` missing here :). Sorry you must be tired of these comments by now!

Reply:
lol :-) I'll do now